### PR TITLE
added 7 motivational quotes

### PR DESCRIPTION
**Pull request by keshari777**
Check one of the below boxes as per your PR

- [:heavy_check_mark:  ] Adding more quotes [issue](https://github.com/avats-dev/motivate/issues/1)
- [ :heavy_check_mark: ] Fixes another issue #16 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ :heavy_check_mark: ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ :heavy_check_mark: ] code is in uniquely-named branch and has no merge conflicts 📁
* [: ] screenshots/GIFs are attached 📎 if applicable
* [ ] ask `@avats-dev` for help, in a comment below

> I'm happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** !

Thanks!

### CODE REVIEW

File: motivate/data/keshari777.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    {
+      "quote": "If opportunity doesn’t knock build a door",
+      "author": "Milton Berle"
+    },
+    {
+      "quote": "If you can’t make a mistake you can’t make anything",
+      "author": "Marva Collin"
+    },
+    {
+      "quote": "It’s not what you do once in a while it’s what you do day in and day out that makes the difference",
```
Comment (avats-dev) [inline motivate/data/keshari777.json:12]: Add a comma after while.

File: motivate/data/keshari777.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "data": [
+    {
+      "quote": "If opportunity doesn’t knock build a door",
+      "author": "Milton Berle"
+    },
+    {
+      "quote": "If you can’t make a mistake you can’t make anything",
```
Comment (avats-dev) [inline motivate/data/keshari777.json:8]: Add a comma after `mistake`.

File: motivate/data/keshari777.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
+{
+  "data": [
+    {
+      "quote": "If opportunity doesn’t knock build a door",
```
Comment (avats-dev) [inline motivate/data/keshari777.json:4]: Add a comma after `knock`.

**Review by avats-dev — CHANGES_REQUESTED** [review]
@keshari777, Update with requested changes. Thanks :+1:

File: motivate/data/keshari777.json (hunk 1 above)
Comment (keshari777) [reply motivate/data/keshari777.json]: updated

File: motivate/data/keshari777.json (hunk 2 above)
Comment (keshari777) [reply motivate/data/keshari777.json]: updated

Comment (keshari777) [reply motivate/data/keshari777.json]: updated

File: motivate/data/keshari777.json (hunk 3 above)
Comment (keshari777) [reply motivate/data/keshari777.json]: updated

**Review by avats-dev — APPROVED** [review]

**Pull request merged**